docs/programming/review.rst

```diff
@@ -229,6 +229,19 @@ Here is a sample command for merging branch "1234-what-was-done"::
 
 It is recommended to define a git alias for `merge --no-commit --squash`.
 
+Conflicts
```

Comment:
I think that this can be created as a section before `Merge your branch`

Maybe rename it to `Keep you branch up to date with main`

```rst
Keep the feature branch up to date
=========================

While waiting for a review or after a review, make sure the feature branch is up to date with the main branch.

We use simple `git merge` to synchronize (sync merge) a feature branch with the main branch.
Don't rebase or rewrite the commit history as this will not play nice with other team members that already have
pulled the branch.

The usual commands are::

    git checkout master
    git pull
    git checkout 1234-the-feature-branch
    git merge master

If there are no conflicts, just commit the changes with default git message.

if there are conflicts, solve the conflicts and merge after all conflicts are solved::

    git merge master
    > git complains about CONFLICT
   > Fix the conflicts using your preferred tool.
    git mergetool
    git commit -a -m "Sync main."
```

Reply:
Maybe add this to `For the person requesting a review` ... but that section is already big.

Maybe we should even consider splitting `For the person requesting a review` into multiple sections.